src/seedu/addressbook/ui/TextUi.java

```diff
@@ -164,5 +165,17 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+        
+    /**
+     * Returns printable strings of each object separated by a newline.
+     */
+    String getPrintableString(Printable... printables){
+        String string = "";
+        for (Printable printable : printables) {
+            string += printable.getPrintableString() + "\n";
```

Comment:
You should use a better delimiter instead of a newline character "\n", else the display of a person's detail is going to be very long inside the UI